README.md

```diff
@@ -13,12 +13,12 @@ Starting a pod with the ops-toolbel timage requires a running Kubelet and health
 #### Running a container locally
 The simplest way of using the `ops-toolbelt` is to just run the following command:
 ```bash
-$ docker run -it eu.gcr.io/gardener-project/gardener/ops-toolbelt-gardenctl:latest
+$ docker run -it eu.gcr.io/gardener-project/gardener/ops-toolbelt:latest
 
 Ubuntu 18.04.2 LTS \n \l
 
 This container comes with the following preinstalled tools:
-curl git tree silversearcher-ag htop less vim tmux bash-completion dnsutils netcat-openbsd iproute2 dstat ngrep tcpdump python-minimal jq yaml2json kubectl pip cat mdv awscli azure-cli python-novaclient python-glanceclient python-cinderclient python-swiftclient google-cloud-sdk aliyun-cli gardenctl
+curl git tree silversearcher-ag htop less vim tmux bash-completion dnsutils netcat-openbsd iproute2 dstat ngrep tcpdump python-minimal jq yaml2json kubectl pip cat mdv awscli azure-cli python-novaclient python-glanceclient python-cinderclient python-swiftclient google-cloud-sdk aliyun-cli
```

Comment:
Just a note: now we do not have a ops-toolbelt containing all iaas clients as that was previously `ops-toolbelt-gardenctl`. Should we build one like that? If not then the IaaS clients should be removed from this line since `ops-toolbelt` doesn't have them installed.

Reply:
good point. As we do not ship gardenctl I doubt that there is much benefit in having these CLIs all installed. I will remove it from the readme. If there is really demand for it we can still create another image containing these, plus maybe also together with gardenctl-v2 and gardenlogin. WDYT?

Reply:
Sounds good. Let's only add the image if there is demand for it.

Reply:
done, also remove the iass-components.yaml
we can restore it from the history in case needed